projects/packages/assets/changelog/fix-jsx-runtime-react-19-polyfill

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Assets: Add JSX runtime polyfill here so that it applies to all relevant plugins in the monorepo, preventing site breakages when WP < 6.6 and packages are included that use React's automatic runtime to transform JSX.
```

Comment:
This seems a little too much for the package's changelog. How about something like this?
```suggestion
Assets: Add JSX runtime polyfill `react-jsx-runtime` for WordPress < 6.6.
```